### DISCUSSION TITLE
https://imagekit.io/blog/automatic-avif-image-optimization-imagekit/

### DISCUSSION THREAD

**Discussion by giscus** [imagekitio/discussions#21] [Blogs]
# https://imagekit.io/blog/automatic-avif-image-optimization-imagekit/

The new AVIF image format offers superior compression and visual quality compared to other formats. Now you can use ImageKit to deliver optimized AVIF images on supported devices.

https://imagekit.io/blog/automatic-avif-image-optimization-imagekit/

**Comment by wizzl0rd** [comment]
2400x3200
JPG • 973.41KB
